src/jest/cloudwatch.test.ts

```diff
@@ -23,20 +25,23 @@ describe('cloudwatch matchers', () => {
     };
     const region = 'region';
     const functionName = 'functionName';
-    const props = { region, function: functionName };
+    const startTime = 12 * 60 * 60 * 1000;
+    const props = { region, function: functionName, startTime };
     const pattern = 'pattern';
 
     beforeEach(() => {
       jest.clearAllMocks();
     });
 
     test('should throw error on filterLogEvents error', async () => {
-      const { verifyProps } = require('../common');
+      const { epochDateMinusHours, verifyProps } = require('../common');
       const { filterLogEvents } = require('../utils/cloudwatch');
 
       const error = new Error('Unknown error');
       filterLogEvents.mockReturnValue(Promise.reject(error));
 
+      epochDateMinusHours.mockReturnValue(11 * 60 * 60 * 1000);
```

Comment:
I think you can remove mocking `epochDateMinusHours` since you're passing `startTime` explicitly